ts/features/bonus/cgn/screens/merchants/CgnMerchantDetailScreen.tsx

```diff
@@ -68,154 +59,129 @@ const CgnMerchantDetailScreen = () => {
       <DiscountListItem item={discount} key={index} />
     ));
 
+  const handlePressMerchantWebsite = (websiteUrl?: string) => {
+    if (websiteUrl !== undefined) {
+      openWebUrl(websiteUrl, () =>
+        IOToast.error(I18n.t("bonus.cgn.generic.linkError"))
+      );
+    }
+  };
   // -------    render
 
+  useHeaderSecondLevel({
+    title: "",
```

Comment:
What about adding the name of the operator at the scroll? I think this should be meant to be the default behaviour by design

Reply:
Addressed into 567df952da2d2b7a04bc2c56483ea0289bd96ebe with the scroll handler with title animated